packages/repository/src/query.ts

```diff
@@ -443,31 +443,30 @@ export class FilterBuilder {
   }
 
   /**
-   * Add a filter object. For conflicting keys with its where object,
-   * create an `and` clause. For any other properties, throw an error.
-   * @param filter filter object
+   * Add an object with open properties. If it is a filter object, create an `and`
```

Comment:
Any reason to loose the type to be `anyObject`? I think we'd better not accept random constraint fields other than the ones in `Filter`.

Reply:
feedback applied